examples/permissioned-token-freezing/test/asset-txfer-test.js

```diff
@@ -46,6 +46,8 @@ describe('Test for transferring asset using custom logic', function () {
     assert.equal(assetDef.clawback, alice.address);
 
     runtime.optIntoASA(assetId, bob.address, {});
+    escrow = undefined;
```

Comment:
why do we need this? In `syncAccounts` we do 
```js
if (escrow) { escrow = runtime.getAccount(escrow.address); }
```

Reply:
escrow has some address loaded somehow.

Reply:
probably when it moves to second test, global value escrow is updated from previous test.